docs/developer-guide/using-layers.md

```diff
@@ -0,0 +1,170 @@
+
+### Accessors
+
+An accessor is a prop that maps an object in `data` to its visual configuration, e.g. the radius of a circle, the color of a line, etc. All accessor prop names start with `get`.
+
+If an accessor prop is set to a function, when the layer is about to be drawn on screen for the first time, the layer will traverse the `data` stream and call the accessor function with each element.
+
+The accessor function receives two arguments:
+
+- `object` - the current element in the data stream. If `data` is an array or an iterable, the element of the current iteration is used. If `data` is a non-iterable object, this argument is always `null`.
+- `objectInfo` (Object) - contextual information of the current element. Contains the following fields:
+  + `index` (Number) - the index of the current iteration
+  + `data` - the value of the `data` prop
+  + `target` (Array) - a pre-allocated array. The accessor function can optionally fill data into this array and return it, instead of creating a new array for every object. In some browsers this improves performance significantly by reducing garbage collection.
+
+The accessor function is typically expected to return either a number or an array.
+
+Some accessors also support constant values instead of functions. When a constant value is provided, it is applied to all objects in the data stream. Unlike a functional value, which is called once for every data object to fill a WebGL buffer proportional to the data size, constant accessors have CPU and memory cost of `O(1)` and are much more performant.
+
+
+### Other Layer Props
+
+The rest of the props are typically numeric or boolean values that apply to the whole layer. These include props that define the render options (opacity, extrusion of the PolygonLayer, font family of the TextLayer, etc.), [coordinate system](/docs/developer-guide/coordinate-systems.md), and [interactivity](/docs/developer-guide/interactivity.md).
+
+
+## Rendering Layers
+
+deck.gl allows you to render multiple layers using the same or different data sets. You simply provide an array of layer instances and deck.gl will render them in order (and handle interactivity when hovering clicking etc).
+
+This allows you to compose visualizations using several primitive layers.
+
+```js
+import {PathLayer, ScatterplotLayer, ArcLayer} from '@deck.gl/layers';
+
+// A layers array contains deck.gl layer instances. It can also be nested, or contain empty objects
+const layers = [
+  new PathLayer({id: 'paths', data: ...}),
+  shouldRenderPoints ? [
+    new ScatterplotLayer({id: 'big-points', data: ...}),
+    new ScatterplotLayer({id: 'small-points', data: ...})
+  ] : null,
+  new ArcLayer({id: 'arcs', data: ...})
+];
+```
+
+```js
+// Use with pure JS
+import {Deck} from '@deck.gl/core';
+const deck = new Deck({...});
+deck.setProps({layers});
+
+// Use with React
+import DeckGL from '@deck.gl/react';
+<DeckGL ... layers={layers} />
+```
+
+## FAQ
```

Comment:
We have a top level FAQ page should we consolidate there and link from here, it seems strange to hide it down here?